ui/vty-lib/src/Ariadne/UI/Vty/Widget/About.hs

```diff
@@ -47,11 +49,11 @@ aboutBanner defAttr selAttr = V.vertCat
   , V.text' defAttr "Ariadne is distributed under the terms of the MPL 2.0 license."
   , V.horizCat
       [ V.text' defAttr "For details, please, see: "
-      , V.text' selAttr "https://serokell.io/ariadne/license"
+      , V.text' selAttr $ Universum.toText licenseUrl
```

Comment:
`Universum.toText` is just `fromString`, no? But why store this as `String` in the first place? I'd use `:: Text` in `UIConfig.hs`